README.md

```diff
@@ -1,7 +1,139 @@
-## ODFE Command Line Interface
+![Build and Test odfe-cli](https://github.com/opendistro-for-elasticsearch/odfe-cli/workflows/Build%20and%20Test%20odfe-cli/badge.svg?branch=main)
+[![codecov](https://codecov.io/gh/opendistro-for-elasticsearch/odfe-cli/branch/main/graph/badge.svg?flag=odfe-cli)](https://codecov.io/gh/opendistro-for-elasticsearch/odfe-cli)
+![PRs welcome!](https://img.shields.io/badge/PRs-welcome!-success)
+# ODFE Command Line Interface
+
+ODFE Command Line Interface (odfe-cli) is an open source tool that lets you manage your Open Distro for Elasticsearch
+cluster from the command line and automate tasks. In addition to standard Elasticsearch operations, you can configure,
+manage, and use the ODFE plugins, such as Alerting, Anomaly Detection, and SQL
+
+odfe-cli is best suited for situations in which you want to quickly combine a few commands, possibly adding them to
+a script for easy access or automation. This example moves a detector "ecommerce-count-qualtity" from staging
+to prod cluster, provided both profiles are available in config file.
+
+```
+odfe-cli ad get ecommerce-count-qualtity --profile stg > ecommerce-count-qualtity.json
+odfe-cli ad create ecommerce-count-qualtity.json --profile prod
+odfe-cli ad start ecommerce-count-qualtity.json --profile prod
+odfe-cli ad stop ecommerce-count-qualtity --profile stg
+odfe-cli ad delete ecommerce-count-qualtity --profile stg
+```
+## Installation:
+
+You can download the binaries directly from the [downloads](https://opendistro.github.io/for-elasticsearch/downloads.html) page
+or from the [releases](https://github.com/opendistro-for-elasticsearch/odfe-cli/releases) section.
+
+
+## Development
+
+### Minimum requirements
+
+odfe-cli shares the same [minimum requirements](https://github.com/golang/go/wiki/MinimumRequirements#minimum-requirements) as Go.
```

Comment:
Docker is also a requirement, right?

Reply:
Docker is a requirement for integration testing only.

Reply:
I updated integration testing topic to include this. @jmazanec15 Can you review ? Thanks.

Reply:
Yeah, but because Minimum requirements is a subsection of development, I think Docker is a requirement for development. I think unit and integration testing should be a component of development. If someone makes a code change, unit and integration tests should be updated/ran in order to validate the change.

That being said, I think line 49 can be removed. What do you think?

Reply:
i removed line 49 and moved docker as requirement to Minimum requirement section.